app/root.tsx

```diff
@@ -125,14 +127,16 @@ export default function App() {
 					<ThemeSwitch />
 				</div>
 				<div className="h-5" />
-				<ScrollRestoration />
-				<Scripts />
+				<ScrollRestoration nonce={nonce}/>
+				<Scripts nonce={nonce}/>
 				<script
+          nonce={nonce}
+          suppressHydrationWarning
```

Comment:
Is this necessary? It should not be.

Reply:
If you're getting that warning then something's wrong. This should all work without any warnings.

Reply:
I was looking on the other project (kentcdodds.com) and this attribute is present. Maybe there is the same problem in the current implementation
I'll try to make it work 💪🏼

Reply:
update: https://react.dev/reference/react-dom/client/hydrateRoot 
`In development mode, React warns about mismatches during hydration.`
I tried with `NODE_ENV=production` and it doesn't complain 👍🏼 so I think I can remove it without concerns